src/mdx.rs

```diff
@@ -1,100 +1,207 @@
+use std::collections::HashMap;
 use std::{env, fs};
-use zed_extension_api::{self as zed, settings::LspSettings};
 
-struct MDXExtension;
+use serde::Deserialize;
+use zed_extension_api::serde_json::json;
+use zed_extension_api::{self as zed, Worktree, serde_json, settings::LspSettings};
 
 const PACKAGE_NAME: &str = "@mdx-js/language-server";
-const SERVER_PATH: &str = "node_modules/.bin/mdx-language-server";
-const SERVER_NAME: &str = "language-server";
+const SERVER_PATH: &str = "node_modules/@mdx-js/language-server/lib/index.js";
```

Comment:
The package-local path is an internal you can’t depend on. The public executable is in `node_modules/.bin/mdx-language-server`.

```suggestion
const SERVER_PATH: &str = "node_modules/.bin/mdx-language-server";
```

Reply:
With that original `SERVER_PATH` the mdx language server was failing since it couldn't find that module. Are you not receiving that error when loading the `mdx-language-server`?

Reply:
I don’t use Zed nor am I familiar with its API nor even Rust. I’m giving this feedback as the MDX language server maintainer and someone who is familiar with Node.js / npm packages.

What does `zed::npm_install_package` do? Does it install locally or globally?

Reply:
It installs the specified NPM package locally.

Reply:
For local installs npm adds executables to `node_modules/.bin`. For global installs npm adds them to `${prefix}/bin`.

Reply:
That's odd, then, that the server was crashing due to failing to find the module. I'll switch the path back locally and test some more to see if I can figure out why.

Reply:
@remcohaszing is correct on this. you can verify this by viewing the zed extension install dir.

Reply:
> The package-local path is an internal you can’t depend on. The public executable is in `node_modules/.bin/mdx-language-server`.

I've hacked on this a bit more, and I cannot get pass the server crashing due to failing to find the executable. When I look in my own `node_modules` directory that path does not exist. Granted in my project I'm testing this one, I'm using Next.js with `@next/mdx` so perhaps the language-server is being hidden away.